backend/python/app/tests/helpers/story_helpers.py

```diff
@@ -42,3 +45,50 @@ def guarantee_story_table_not_empty(db):
         )
         db.session.add(obj)
         assert db.session.commit() == None
+
```

Comment:
oh I think lynn and jenn talked about implementing something like this, i'll share this pr with them